src/js/providers/providers.js

```diff
@@ -90,16 +90,21 @@ define([
                 var html5Idx = _.indexOf(providers, _.findWhere(providers, {name: 'html5'}));
                 providers.splice(html5Idx, 0, flashProvider);
             }
+            if (hlsjsdefault) {
+                var hlsIndex =  _.indexOf(providers, _.findWhere(providers, {name: 'hlsjs'}));
+                var hlsjsProvider = providers.splice(hlsIndex, 1)[0];
+                providers.unshift(hlsjsProvider);
```

Comment:
We can just insert hlsjs before html5 in commercial. The supports check won't let ungated Browsers use it.